Assets/Plugins/Android/Core/AndroidPlatformSpecifics.cs

```diff
@@ -31,7 +33,7 @@
 using System.Diagnostics;
 
 #if UNITY_ANDROID && !UNITY_EDITOR
-[assembly: AlwaysLinkAssembly]
+
```

Comment:
Instead of applying this assembly attribute within a conditional within a specific platform - it was implemented once in the baser `PlatformSpecifics` abstract class.

Reply:
It does indeed work properly - I had to test the different build configurations running around my parents dining room between my windows laptop, my mac mini, and the linux vm I had running on one of them to make sure all the build configurations worked properly. 

I think typically this kind of thing is usually resolved with an `AssemblyInfo.cs` file, but Unity doesn't typically feature those.

Reply:
Just to double check, these are tested with IL2CPP code stripping enabled?
That was the conditions when it was causing trouble

Reply:
Yes, it appears to be working in that circumstance as well.